examples/Bunch/include/simulation_defines/param/species.param

```diff
@@ -71,6 +71,7 @@ typedef currentSolver::Esirkepov<UsedParticleShape> UsedParticleCurrentSolver;
  *
  * - particles::pusher::Vay : better suited relativistic boris pusher
  * - particles::pusher::Boris : standard boris pusher
+ * - particles::pusher::ReducedLandauLifshitz: propagate with radiation reaction
```

Comment:
since this is not based on a "momentum-conserving" solver as we are used to have, writing
`4th order RungeKutta pusher with classical radiation reaction` might be more clear?